.env.example

```
@@ -3,4 +3,7 @@ NEXT_PUBLIC_SUPABASE_URL=your-supabase-url
 NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY=your-supabase-publishable-key
 SUPABASE_SERVICE_ROLE_KEY=your-supabase-service-role-key
 DATABASE_URL="postgresql://postgres.cmvqbjbohfloobgrcdoo:your-password@aws-1-ap-northeast-1.pooler.supabase.com:6543/postgres?pgbouncer=true"
-DIRECT_URL="postgresql://postgres.cmvqbjbohfloobgrcdoo:your-password@aws-1-ap-northeast-1.pooler.supabase.com:5432/postgres"
\ No newline at end of file
+DIRECT_URL="postgresql://postgres.cmvqbjbohfloobgrcdoo:your-password@aws-1-ap-northeast-1.pooler.supabase.com:5432/postgres"
+
+# CSVインポートで作成されるユーザーのデフォルトパスワード
+DEFAULT_USER_PASSWORD=P@ssw0rd
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)